samples/apps/copilot-chat-app/deploy/deploy-webapp.sh

```diff
@@ -131,6 +122,14 @@ if [[ "$CORS_RESULT" == "null" ]]; then
     az webapp cors add --name $WEB_API_NAME --resource-group $RESOURCE_GROUP --subscription $SUBSCRIPTION --allowed-origins $origin
 fi
 
+echo "Updating redirect URI in AAD app registration to '$ORIGIN'..."
+eval OBJECT_ID=$(az ad app show --id $APPLICATION_ID | jq '.id')
+az rest \
+    --method PATCH \
+    --uri "https://graph.microsoft.com/v1.0/applications/$OBJECT_ID" \
+    --headers 'Content-Type=application/json' \
+    --body "{spa:{redirectUris:['$ORIGIN']}}"
```

Comment:
This command doesn't seem to be working, trying to figure out what's wrong... no errors, but i'm not seeing the app registration updating. The object id is parsed correctly.